### ISSUE TITLE
infer code

### ISSUE THREAD

**Issue by huangyangke** [mlzxy/devit#26]
May I ask if there is any inference code provided and I would like to know the inference code for the feed shot.

**Comment by YELKHATTABI** [comment]
I don't know if I used everything correctly but the steps would be the following : 
- You start by converting your dataset into a coco format dataset
- You use https://github.com/mlzxy/devit/blob/main/tools/extract_instance_prototypes.py to extract prototypes, but for this you need to add you dataset here : https://github.com/mlzxy/devit/blob/main/detectron2/data/datasets/builtin.py
- Once your prototypes are computed you need to average each class and normalize it, and add a the class names as a list under the key 'label_names'
- Once you have done this, you can use demo/demo.py and changing category_space in the main function (you put the images on which you want to make inferences in the input dir, and you get the output in the output dir)

tldr: you can start from demo/demo.py
